jwt/keyset.go

```diff
@@ -238,7 +237,7 @@ func unmarshalResp(r *http.Response, body []byte, v interface{}) error {
 	ct := r.Header.Get("Content-Type")
 	mediaType, _, parseErr := mime.ParseMediaType(ct)
 	if parseErr == nil && mediaType == "application/json" {
-		return fmt.Errorf("got Content-Type = application/json, but could not unmarshal as JSON: %v", err)
```

Comment:
Maybe I should leave a comment here on why we don't return the error. Something like

```
We don't return the error to prevent leaking data
```

Reply:
why are we worried about returning error data from parsing the public keyset?